python/cucim/tests/fixtures/testimage.py

```diff
@@ -16,7 +16,7 @@
 import shutil
 
 import pytest
-from pytest_lazyfixture import lazy_fixture
+from pytest_lazy_fixtures import lf as lazy_fixture
```

Comment:
nice to see that this can continue to be used as-is without further changes to the test cases using the fixture